frontend/freedom-fighters/style.css

```
@@ -0,0 +1,240 @@
+body.light-theme {
+  --rulers-bg-dark: #f4f6f9;
+  --rulers-card-dark: #ffffff;
+  --rulers-border-dark: rgba(230, 81, 0, 0.2);
+  --rulers-text-light: #212529;
+  --rulers-text-muted: #6c757d;
+}
+
+.rulers-main, .ruler-profile-page {
+  background-color: var(--rulers-bg-dark);
+  color: var(--rulers-text-light);
+  font-family: 'Outfit', sans-serif;
```

Stylelint (17.14.0)

[error] Expected no quotes around "Outfit" (font-family-name-quotes)

(font-family-name-quotes)